inference-engine/tests/functional/inference_engine/serialization/single_layer/group_convolution_backprop_data.cpp

```diff
@@ -16,11 +16,7 @@ TEST_P(GroupConvBackpropDataLayerTest, Serialize) {
 
 const std::vector<InferenceEngine::Precision> precisions = {
     InferenceEngine::Precision::FP64, InferenceEngine::Precision::FP32,
-    InferenceEngine::Precision::FP16, InferenceEngine::Precision::BF16,
-    InferenceEngine::Precision::I8,   InferenceEngine::Precision::I16,
-    InferenceEngine::Precision::I32,  InferenceEngine::Precision::I64,
-    InferenceEngine::Precision::U8,   InferenceEngine::Precision::U16,
-    InferenceEngine::Precision::U32,  InferenceEngine::Precision::U64,
+    InferenceEngine::Precision::FP16, InferenceEngine::Precision::BF16
```

Comment:
According to [spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/GroupConvolutionBackpropData_1.md), for which inputs are of floating point type, instances of this op with integer element type are removed for GroupConvBackpropDataLayerTest.